cypress.json

```diff
@@ -7,6 +7,7 @@
     "videosFolder": "tests/cypress/videos",
     "supportFile": "tests/cypress/support",
     "projectId": "tt5hbz",
+    "defaultCommandTimeout": 10000,
```

Comment:
This didn't resolve the flakes overall (I was able to still get flakes on repeated reruns with  the form navigation) but I don't think it hurts to leave it. We can see if this slows down the CI time at all and assuming it doesn't, seems fine to me.